src/gamepad/standard-gamepad-axis.vala

```diff
@@ -0,0 +1,38 @@
+// This file is part of GNOME Games. License: GPLv3
+
+// This file is part of GNOME Games. License: GPLv3
+
+public enum LibGamepad.StandardGamepadAxis {
+	/**
+	 * The horizontal axis of the left stick.
+	 *
+	 * Negative is left, positive is right.
+	 */
+	LEFT_X,
+
+	/**
+	 * The vertical axis of the left stick.
+	 *
+	 * Negative is up, positive is down.
+	 */
+	LEFT_Y,
+
+	/**
+	 * The horizontal axis of the right stick.
+	 *
+	 * Negative is left, positive is right.
+	 */
+	RIGHT_X,
+
+	/**
+	 * The vertical axis of the right stick.
+	 *
+	 * Negative is up, positive is down.
+	 */
+	RIGHT_Y,
+
+	/**
+	 * An unknown axis
+	 */
+	UNKNOWN,
```

Comment:
What do you think of moving UNKNOWN to the top si it's value is 0? This way the default value will be UNKNOWN and compare to 0 and false, which is probably more sane.